src/middlewared/middlewared/etc_files/local/ssh/config.py

```diff
@@ -24,6 +24,10 @@ def generate_ssh_config(middleware):
         if os.path.exists(os.path.join(SSH_CONFIG_PATH, f)):
             os.chmod(os.path.join(SSH_CONFIG_PATH, f), 0o600)
 
+    twofactor_auth = middleware.call_sync('auth.twofactor.config')
+    if not twofactor_auth['enabled'] or not twofactor_auth['services']['ssh']:
+        os.remove('/usr/local/etc/users.oath')
```

Comment:
Can't we invent some mechanism that will prevent this file from being created instead?